rpcclient.go

```diff
@@ -655,8 +657,8 @@ func roundIndex(start, max int) (result []int) {
 	return
 }
 
-// IsNetworkError will decide if an error is network generated or RPC one
-func IsNetworkError(err error) bool {
+// IsConnectionOrServiceErr will decide if an error is network generated or RPC one
+func IsConnectionOrServiceErr(err error) bool {
```

Comment:
Please break the function in two separate ones, so we can properly understand the context.
Thanks!